.github/workflows/publish_dockerhub_main.yml

```diff
@@ -0,0 +1,53 @@
+# TODO: trigger only if testing or any other checks previously succeeded
+name: Push to DockerHub (main)
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: autoinstrumentation-ebpf
```

Comment:
This is verbose. Can we shorten this to the same name we plan to use for any binary build?

Reply:
Sure. I observed there are many `autoinstrumentation-*` images in DockerHub so I wanted to follow the same pattern. But I'm fine using something shorter as `ebpf-instrument` (same as the binary name). Is it Ok to you?

Reply:
I'm in favor of `ebpf-instrument`.